Hestia/app/src/androidTest/java/com/rugged/application/hestia/backend/exceptions/ServerExceptions/DataBaseExceptionTest.java

```diff
@@ -0,0 +1,35 @@
+package com.rugged.application.hestia.backend.exceptions.ServerExceptions;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
```

Comment:
Android Studio creates redundant white spaces in the imports. These are really ugly and unnecessary, in my opinion. I believe we should remove them. What do others think?